velox/exec/tests/AggregationTest.cpp

```diff
@@ -752,6 +752,29 @@ TEST_F(AggregationTest, allKeyTypes) {
       " GROUP BY c0, c1, c2, c3, c4, c5");
 }
 
+TEST_F(AggregationTest, dateAggregateTest) {
```

Comment:
There is a dedicated test file for min and max test: MinMaxTest.cpp. Consider moving the new test there and using testAggregations helper function for better test coverage.

Reply:
Sure, thank you